18_abandoned_isles.conf

```
@@ -0,0 +1,14 @@
+team.2.color = blue
+team.2.pos = (86,-5,-101)
+team.1.pos = (-79,-5,91)
+team.1.color = red
+team.1 = red
+author = gimp, caoutiouspotato
+rotation = x
+h = 140
+team.2 = blue
+name = Abandoned Isles
+initial_stuff = default:sword_steel,default:pick_steel,shooter:pistol,default:sandstone 50
+r = 115
+screenshot = 18_abandoned_isles.png
+license = CC-BY-4.0
```

Hyphens should be removed: https://creativecommons.org/licenses/by/4.0/deed.ast

```suggestion
license = CC BY 4.0
```